client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,101 @@
+  const classes = useStyles();
+
+  const { loggedInUser } = useAuth();
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid container className={classes.cards}>
+        <Grid item className={classes.profileCard}>
+          <Card>
+            <CardActionArea>
+              <CardMedia
+                className={classes.media}
+                image="https://images.unsplash.com/photo-1570129477492-45c003edd2be?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80"
+              />
+              <CardContent className={classes.cardContent}>
+                <Avatar className={classes.avatar} />
+                <Typography className={classes.nameField}>First Last</Typography>
+                <Typography className={classes.subtextField}>Loving pet sitter</Typography>
+                <IconButton className={classes.locationField} disabled>
+                  <LocationOnIcon color="primary" fontSize="small" />
+                  Location
+                </IconButton>
+                <Typography className={classes.aboutMeField}>About me</Typography>
+                <Typography>
+                  Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum
+                  Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum
+                </Typography>
+                <Box className={classes.imagesBox}>
+                  <ProfileImageList />
+                </Box>
+              </CardContent>
+            </CardActionArea>
+          </Card>
+        </Grid>
+        <Grid item className={classes.requestCard}>
+          <Card>
+            <CardActionArea>
+              <CardContent>
+                <Typography className={classes.rphField}>$RATE/hr</Typography>
+                <Rating name="read-only" readOnly className={classes.ratingStars} />
+                <form>
+                  <TextField
+                    id="startDate"
+                    label="Start Date"
+                    type="datetime-local"
+                    InputLabelProps={{
+                      shrink: true,
+                    }}
+                    variant="outlined"
+                    className={classes.datePicker}
+                  />
+                  <TextField
+                    id="endDate"
+                    label="End Date"
+                    type="datetime-local"
+                    InputLabelProps={{
+                      shrink: true,
+                    }}
+                    variant="outlined"
+                    className={classes.datePicker}
+                  />
+                </form>
+              </CardContent>
+            </CardActionArea>
+            <CardActions>
+              <Button className={classes.requestButton} size="large" color="primary" variant="contained">
+                Send Request
+              </Button>
```

Comment:
This button should probably be of `type='submit'` and be wrapped within the form component. It might be a good idea to use `component='form'` on the appropriate `Card` component/subcomponent instead of providing a separate JSX element.